src/main/java/io/fabric8/maven/docker/config/BuildImageConfiguration.java

```diff
@@ -235,6 +240,10 @@ public BuildTarArchiveCompression getCompression() {
         return compression;
     }
 
+    public Map<String, String> getQueryArgs() {
```

Comment:
Maybe we can call it some what less technical (e.g. how it is used) but e.g. 'options' (or 'buildOptions') ?